panel/src/components/Sections/ModelsSection.vue

```diff
@@ -231,6 +309,24 @@ export default {
 			this.searching = !this.searching;
 			this.searchterm = null;
 		},
+		onSelect(event, item) {
+			if (event.target.checked) {
```

Comment:
`Todo:` Making this depend on the event (and more specifically a checkbox event) that many layers up from the actual input seems not ideal to me. Can this just accept the item as parameter and decide what to do based on whether the item is already in the selected array or not?

Reply:
You are right. I've refactored it to only send the item and item index.